### PR TITLE
removed transfer callbacks

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: raiden/raiden_service.py [hunk 1]
```diff
@@ -511,17 +485,19 @@ def expect_exchange(
...
+        `token_address` and/or `partner_address`.
+
         Args:
             token_address (bin): an optionally provided token address
             partner_address (bin): an optionally provided partner address
         Return:
-            A list containing all channels the node participates. Optionally filtered by an token address
-            and/or partner address.
+            A list containing all channels the node participates. Optionally
+            filtered by an token address and/or partner address.
```
Comment (konradkonrad) [inline raiden/raiden_service.py]: `s/an/a/` :P

File: raiden/transfermanager.py [hunk 2]
```diff
@@ -210,31 +171,25 @@ def _direct_or_mediated_transfer(self, amount, identifier, direct_channel, callb
...
             async_result = self.tokenmanager.raiden.protocol.send_async(
                 direct_channel.partner_state.address,
                 direct_transfer,
             )
             return async_result
 
-    def _mediated_transfer(self, amount, identifier, target, callback):
-        asunc_result = AsyncResult()
+    def _mediated_transfer(self, amount, identifier, target):
+        async_result = AsyncResult()
```
Comment (konradkonrad) [inline raiden/transfermanager.py]: :)

**Comment by LefterisJP** [comment]
@konradkonrad We had a chat with @hackaugusto about this.

The reasoning is that these were added by max with the idea that they will be used by some kind of future python library API. This is not happening any time soon, and as things stand right now they are more of a bother as they create multiple points where state changes can occur. 

For restartable tasks and the state machine we want to absolutely minimize those.

File: raiden/raiden_service.py [hunk 3]
```diff
@@ -511,17 +485,19 @@ def expect_exchange(
...
         Raises:
-            UnknownTokenAddress: An error occurred when the token address is unknown to the node.
-            KeyError: An error occurred when the given partner address isn't associated with the given
-            token address.
+            UnknownTokenAddress: An error occurred when the token address is
+            unknown to the node.  KeyError: An error occurred when the given
```
Comment (LefterisJP) [inline raiden/raiden_service.py]: Enter a newline after `to the node`

File: raiden/transfermanager.py (hunk 2 above)
Comment (LefterisJP) [inline raiden/transfermanager.py]: ^_^

**Pull request merged**